api/v1alpha3/azurecluster_types.go

```diff
@@ -19,7 +19,7 @@ package v1alpha3
 import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha4"
```

Comment:
why in the v1alpha3 we set to use the cluster-api v1alpha4 and not v1alpha3?

Reply:
you are correct, will fix it